src/libraries/System.Text.RegularExpressions/gen/RegexGenerator.Emitter.cs

```diff
@@ -299,8 +342,29 @@ static void AppendHashtableContents(IndentedTextWriter writer, Hashtable ht)
             }
         }
 
-        /// <summary>Emits the body of the FindFirstChar override.</summary>
-        private static RequiredHelperFunctions EmitFindFirstChar(IndentedTextWriter writer, RegexMethod rm, string id)
+        private static void EmitScan(IndentedTextWriter writer, RegexMethod rm, string id)
+        {
```

Comment:
That said, this method is tiny and there's little-to-no customization of it.  The function could just be deleted and its code moved into the call site.

Reply:
Yeah, I believe this is what I was suggesting offline, the only customization is whether or not we should call CheckTimeout depending on if there is one or not. If we prefer to not have to generate one more method, I'm fine with keeping Scan inside regex dll and only emit FFC and Go (with new names) for now. This is how I originally had it but I believe you asked to instead move to generate Scan instead in favor of a future move away from the FFC/Go design.

Reply:
I very well may have :)  It's fine as-is.